vochain/indexer/indexer.go

```diff
@@ -493,8 +493,13 @@ func (idx *Indexer) Rollback() {
 func (idx *Indexer) OnProcess(pid, eid []byte, censusRoot, censusURI string, txIndex int32) {
 	idx.lockPool.Lock()
 	defer idx.lockPool.Unlock()
-	data := &indexertypes.IndexerOnProcessData{EntityID: eid, ProcessID: pid}
-	idx.newProcessPool = append(idx.newProcessPool, data)
+	if err := idx.newEmptyProcess(pid); err != nil {
```

Comment:
this one _does_ need to use `blockTx`, otherwise we're writing the process to the database right away without waiting for the call to `Indexer.Commit`.

Reply:
Removed